cmd/glasskube/cmd/purge.go

```diff
@@ -0,0 +1,64 @@
+package cmd
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/glasskube/glasskube/internal/clicontext"
+	"github.com/glasskube/glasskube/internal/cliutils"
+	"github.com/glasskube/glasskube/internal/config"
+	"github.com/glasskube/glasskube/internal/util"
+	"github.com/glasskube/glasskube/pkg/bootstrap"
+	"github.com/glasskube/glasskube/pkg/purge"
+	"github.com/glasskube/glasskube/pkg/statuswriter"
+	"github.com/spf13/cobra"
+)
+
+type purgeOptions struct {
+	yes bool
+}
+
+var purgeCmdOptions = purgeOptions{}
+
+var purgeCmd = &cobra.Command{
+	Use:   "purge",
+	Short: "Purge Glasskube from a Kubernetes cluster",
+	Long: "Purges Glasskube from a Kubernetes cluster, " +
+		"thereby uninstalling the Glasskube operator and all related resources.",
+	Args:   cobra.NoArgs,
+	PreRun: cliutils.SetupClientContext(false, util.Pointer(true)),
+	Run: func(cmd *cobra.Command, args []string) {
+		cfg, _ := cliutils.RequireConfig(config.Kubeconfig)
+		client := purge.NewPurger(cfg)
+		ctx := cmd.Context()
+
+		client.WithStatusWriter(statuswriter.Spinner())
```

Comment:
Should honor `--no-progress` flag
```suggestion
		if !rootCmdOptions.NoProgress {
			client.WithStatusWriter(statuswriter.Spinner())
		}
```